docs/books/user-guide/routing.adoc

```diff
@@ -551,6 +551,88 @@ autoLink {
 For information about additional attributes, see link:{qdrouterdConfManPageUrl}#_autolink[autoLink] in the `qdrouterd.conf` man page.
 --
 
+=== Handling Undeliverable Messages for an Address
+
+You handle undeliverable messages for an address by configuring _fallback destinations_.
+A fallback destination is a queue on a broker that stores messages that are not directly routable to any consumers.
+
+During normal message delivery,
+{RouterName} delivers messages to the consumers that are attached to the router network.
+However, if no consumers are reachable,
+the messages are diverted to any fallback destinations that were configured for the address.
+When a consumer reconnects and becomes reachable again,
+it receives the messages stored at the fallback destination.
+
+[NOTE]
+====
+{RouterName} does not preserve delivery order for messages stored at a fallback destination. When a consumer reconnects, it will receive the stored messages interleaved with any new messages sent by producers.
+====
+
+.Prerequisites
+
+* The router is connected to a broker.
++
+For more information, see xref:connecting-to-external-amqp-containers-router[].
+
+.Procedure
+
+This procedure enables fallback for an address
+and configures autolinks to connect to the broker queue
+that provides the fallback destination for the address.
+
+. Enable fallback destinations for the address.
```

Comment:
Change the period to a colon? (I realize this is a stylistic element that might conflict with the rest of the docs)